cluster/validate-cluster.sh

```diff
@@ -102,4 +102,10 @@ for (( i=0; i<${#MINION_NAMES[@]}; i++)); do
       sleep 30
     done
 done
+
+# Here an individual provider can add extra tests, 
+# For example 'vagrant ssh master1 kubectl get pods' should work
+# to show existing pods, and so on.
+${KUBE_ROOT}/cluster/${KUBERNETES_PROVIDER}/validate-provider.sh
```

Comment:
You would need a shell file for each provider?

My bias is we do not do this piece since the goal is to move more of this into a Go program.